imports/server/migrations/43-puzzle-activity.ts

```diff
@@ -0,0 +1,21 @@
+import ConsolidatedActivities from '../models/ConsolidatedActivities';
+import RecentActivities from '../models/RecentActivities';
+import Migrations from './Migrations';
+
+Migrations.add({
+  version: 43,
+  name: 'Indexes for puzzle activity tracking',
+  async up() {
+    await RecentActivities.createIndexAsync({
+      ts: 1,
+      type: 1,
+      puzzle: 1,
```

Comment:
Any reason you left `hunt` out of the unique index here? Practically, I imagine it doesn't /really/ matter since puzzle IDs are expected to be unique, but I couldn't tell if this was intentional and if there's a performance or size or correctness reason to include it or not.

Reply:
Hmm. I think in general when I have unique indexes I prefer for them to be minimally defined (i.e. not include fields that don't contribute to the uniqueness), so my instinct would be to drop it from the `ConsolidatedActivities` index.

We'll need another index when we get around to querying anyway, since we need an index that starts with `hunt`

Reply:
Thanks for the explanation and guidelines!